example/panel.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <style>
+      body {
```

Comment:
Maybe let's just give `body, html` a `padding: 0` for good measure?  (and eg a margin on the button). i expect anybody will want to do that, so the html panel behaves pretty much like a div.

Reply:
Not sure what you mean here; the buttons already have `margin: 0.6rem auto`